pjsip-apps/src/swig/csharp/pjsua2maui/README.md

```diff
@@ -0,0 +1,126 @@
+# pjsua2maui - pjsip on .net maui 
+
+## STEPS FOR BUILDING:
+
+### Android
+
+- SETUP:
+    - Visual Studio Code & .NET MAUI - [Install steps](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-11.0&tabs=visual-studio-code)
+    - Java 17 
+    - Android NDK r21d
+    - pjsip v2.14.1
+
+-  Run the following commands:
+
+```bash 
+    export ANDROID_SDK_ROOT=/somewhere/there/is/installed/the/sdk/
+    sdkmanager --install "ndk;28.0.12674087" --channel=3 --sdk_root=$ANDROID_SDK_ROOT
+    sdkmanager --install "cmake;3.31.0" --sdk_root=$ANDROID_SDK_ROOT
+    sdkmanager --install "build-tools;35.0.0" "platform-tools" "platforms;android-35" --sdk_root=$ANDROID_SDK_ROOT
+    echo yes | sdkmanager --licenses --sdk_root=$ANDROID_SDK_ROOT
```

Comment:
Add note:
If you are sharing the Android SDK location with Android Studio, the recommended way to install/update the SDK and accept licenses is by using Android Studio.